pkgs/by-name/db/dbd/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "dbd";
+  version = "1.50-unstable-2016-01-04";
+
+  src = fetchFromGitHub {
+    owner = "gitdurandal";
+    repo = "dbd";
+    rev = "8cf5350781b6753fcdd863148a5dcc6976e693ca";
+    hash = "sha256-b2yBZ2/Ab+SviKNlyZgdfiZ7GGZ1sonZnblD0i+vuFw=";
+  };
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+    "CC=${stdenv.cc.targetPrefix}cc"
+  ];
+
+  buildFlags = lib.optionals stdenv.isDarwin [ "darwin" ] ++ lib.optionals (stdenv.hostPlatform.isUnix && !stdenv.isDarwin) [ "unix" ];
+
+  meta = with lib; {
+    description = "A Netcat-clone, designed to be portable and offer strong encryption";
```

Convention is to remove leading `A`'s
```suggestion
    description = "Netcat-clone, designed to be portable and offer strong encryption";
```